src/modules/hub.bicep

```diff
@@ -70,3 +81,6 @@ output location string = location
 
 @description('Resource ID of the storage account created for the hub instance. This must be used when creating the Cost Management export.')
 output storageAccountId string = storageAccount.outputs.resourceId
+
+@description('The Resource ID of the Data factory.')
+output dataFactoryId string = dataFactory.outputs.resourceId
```

Comment:
Does the user need the data factory ID for anything? If not, I wouldn't pass it thru to outputs. It's easier to add it than take it away. If we need to keep it, we should update the docs as well.

Reply:
removed